src/components/Home.jsx

```diff
@@ -1,10 +1,41 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { collection, addDoc } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+
+import { getToken } from '@the-collab-lab/shopping-list-utils';
 
 function Home() {
+  const navigate = useNavigate();
+
+  //check for existing token and go to List View if it is there
+  useEffect(() => {
+    const checkToken = localStorage.getItem('list-token');
+    if (checkToken) navigate('/listView');
+  }, [navigate]);
+
+  const handleNewList = async () => {
+    const token = getToken();
+    localStorage.setItem('list-token', token);
+
+    //add token to Firestore
+    try {
+      const docRef = await addDoc(collection(db, 'token'), {
+        token,
+      });
+      navigate('/listView');
```

Comment:
the navigation is SO user friendly to bypass the create list page if the user has a token in local storage